### DISCUSSION TITLE
Adjusting table scroll bar width

### DISCUSSION THREAD

**Discussion by runrc** [cogentcore/core#1129] [Q&A] [answered]
The scrollbar on the right side of the table is very small/thin. 
How can I make the scrollbar width bigger?

**Comment by kkoreilly** [comment]
Are you referring to the scrollbar width or handle height? The handle height of the scrollbar is based on the amount of content with a lower bound, but we could increase that minimum value, which would increase the scrollbar handle height. You can change the width of the scrollbar for a single table with this code (I will change this to be easier now by making the value inherited):

```go
	tree.AddChildInit(tb, "grid", func(w *core.ListGrid) {
		w.Styler(func(s *styles.Style) {
			s.ScrollBarWidth.Dp(50) // or any other value
		})
	})
```

Or for all scrollbars with this code:
```go
	styles.ScrollBarWidthDefault = 50 // or any other value
```

**Reply by runrc** [reply]
Yes, thank you I was referring to the width of the scroll bar. This resolves the issue, although it might be worthwhile adding this to the Styling section of the documentation.

**Reply by kkoreilly** [reply, marked as answer]
OK. In answering this I realized that this structure could be improved, so after #1132 is merged, it will be this instead:

For a single table/frame/widget and all of its children:

```go
	tb.Styler(func(s *styles.Style) {
		s.ScrollbarWidth.Dp(50)
	})
```

For all scrollbars:

```go
	styles.DefaultScrollbarWidth.Dp(50)
```

Apologies for the change.

**Reply by runrc** [reply]
That's great, it's not a problem at all. It's important these things are ironed out and documented. Thanks!

**Reply by kkoreilly** [reply]
OK, I also improved the styling documentation in 7aef3189e6436f928f9fc1e6cec2c65b8698161e with a link to a list of all styling properties, including the scrollbar width, which now also has struct field documentation mentioning `DefaultScrollbarWidth`.
